common/persistence/sql/shard.go

```diff
@@ -137,6 +137,12 @@ func (m *sqlShardStore) UpdateShard(
 	})
 }
 
+func (m *sqlShardStore) CloseShard(
```

Comment:
does this PR need to include an implementation for Cassandra, MySQL, etc.?  Or does this line provide a default impl?